management/backup/locations_test.go

```diff
@@ -95,7 +96,6 @@ func TestAddWrongLocation(t *testing.T) {
 		pmmapitests.AssertAPIErrorf(t, err, 400, codes.InvalidArgument, "Missing location config.")
 		assert.Nil(t, resp)
 	})
-
```

Comment:
I personally prefer to have these separate lines

Reply:
+1